doc/threat_model/Untrusted_Capsule_Isolation.md

```diff
@@ -0,0 +1,23 @@
+Untrusted Capsule Isolation
+===========================
+
+## Isolation Mechanism
+
+Untrusted capsules are limited to what they should be able to access within
+Rust's type system without using `unsafe`. That isolation is implemented by
+banning `unsafe` from use in untrusted capsule code. This isolation is
```

Comment:
```suggestion
Rust's type system without using `unsafe`. That isolation is implemented by
banning `unsafe` from use in untrusted capsule code and any libraries the untrusted capsule
uses (except for the Rust core library). However, there is currently no automated mechanism
for enforcing this restriction in libraries, and therefore Tock prohibits using any external libraries.
This isolation is
```

Reply:
I didn't realize that untrusted capsules were limited in what libraries they can use. I adjusted this paragraph to indicate that limitation.